rotkehlchen/api/rest.py

```diff
@@ -4499,7 +4513,7 @@ def _query_kraken_staking_events(
                 ),
                 'received': [
                     {
-                        'asset': entry[0].identifier,
```

Comment:
Why this change?

Reply:
I saw that we didn't need to create an Asset object since we were taking directly the identifiers. This saves us from creating useless objects